config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
 devise_for :users
 
-root 'welcome#index'
+ root 'welcome#index'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.